src/creatures/players/imbuements/imbuements.h

```diff
@@ -21,6 +21,7 @@
 #define SRC_CREATURES_PLAYERS_IMBUEMENTS_IMBUEMENTS_H_
 
 #include "declarations.hpp"
+#include "creatures/players/imbuements/imbuement.h"
```

Comment:
should this file also be added into this PR?
It doesn't seems related to the recursive search